scripts/replace_simlink.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
this is not really related, but as useful script i will keep this into this commit.